core/src/main/java/site/ycsb/Client.java

```diff
@@ -252,11 +252,21 @@ private static void exportMeasurements(Properties props, int opcount, int warmup
         exporter = new TextMeasurementsExporter(out);
       }
 
-      exporter.write("OVERALL", "RunTime(ms)", runtime);
-      double throughput = 1000.0 * (opcount) / (runtime);
-      exporter.write("OVERALL", "Throughput(ops/sec)", throughput);
-      exporter.write("OVERALL", "Operations(payload)", opcount);
-      exporter.write("OVERALL", "Operations(warm-up)", warmupopcount);
+      long warmupRuntime = warmupEnd - warmupStart;
+      long payloadRuntime = payloadEnd - payloadStart;
+      long overallRuntime = warmupRuntime + payloadRuntime;
+      double warmupThroughput = 1000.0 * (warmupOpsCount) / (warmupRuntime);
+      double payloadThroughput = 1000.0 * (payloadOpsCount) / (payloadRuntime);
```

Comment:
Do we need brackets here?
```suggestion
      double warmupThroughput = 1000.0 * warmupOpsCount / warmupRuntime;
      double payloadThroughput = 1000.0 * payloadOpsCount / payloadRuntime;
```

Reply:
No, we don't. Removed. It's just leftovers from YCSB origins.